docs/source/api_reference.rst

```diff
@@ -26,20 +26,6 @@ Deep Feature Synthesis
 
     dfs
 
-
```

Comment:
i think we can keep this in like we do for the nlp primitives. we just need to mention how to install

https://docs.featuretools.com/en/stable/api_reference.html#natural-language-processing-primitives